pkg/api/admin_users.go

```diff
@@ -67,7 +68,11 @@ func AdminUpdateUserPassword(c *models.ReqContext) response.Response {
 	if err := web.Bind(c.Req, &form); err != nil {
 		return response.Error(http.StatusBadRequest, "bad request data", err)
 	}
-	userID := c.ParamsInt64(":id")
```

Comment:
The previous code reads cleaner. Could perhaps the method be changed instead? I'm not sure I fully understand the motivation behind the change, perhaps the method returns an error or something like this?

Reply:
That is because the implicit read no numerical query variable into 0, it is a potential security issue, and since it is just one line of code, we decided not to wrap it into functions, instead we just use standard library.